### ISSUE TITLE
How can I add an checkbox to promt and catch result?

### ISSUE THREAD

**Issue by art1sh0k** [fabien-d/alertify.js#269]
I think, it will be good additional to this framework. Can you help me?

**Comment by MincDev** [comment]
I agree with this, sometimes we require a "Dont show this again" checkbox in the popup. Would really appreciate some help on this as well.

**Comment by MincDev** [comment]
@art1sh0k If you require a checkbox in the prompt, simply do the following after the alertify code:

$(".ajs-content").append("\<input type="checkbox" \/\>");
